cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -401,6 +401,9 @@ function (google_cloud_cpp_enable_features)
             if (NOT ("storage" IN_LIST GOOGLE_CLOUD_CPP_ENABLE))
                 add_subdirectory(google/cloud/storage)
             endif ()
+        elseif ("${feature}" STREQUAL "storage")
+            add_subdirectory(protos/google/cloud/storage/tests)
+            add_subdirectory(google/cloud/storage)
```

Comment:
I think this stops building `google/cloud/storage/examples`

Reply:
Right you are.  Fixed.